langserver/build_context.go

```diff
@@ -42,35 +41,29 @@ func (h *HandlerShared) OverlayBuildContext(ctx context.Context, orig *build.Con
 	ctxt := &copy
 
 	ctxt.OpenFile = func(path string) (io.ReadCloser, error) {
-		return fs.Open(ctx, path)
+		return fs.Open(ctx, vfsPath(path))
```

Comment:
why does this wrapper need to exist? Shouldn't users of build context only be using virtual paths?